pkgs/development/libraries/sokol/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation {
+  pname = "sokol";
+  version = "unstable-2022-06-13";
+
+  src = fetchFromGitHub {
+    owner = "floooh";
+    repo = "sokol";
+    rev = "3c7016105f3b7463f0cfc74df8a55642e5448c11";
+    sha256 = "1dkh65n3r1i97yr6b05prmg1s64yb23275v5p57s5rg9ckldp8bl";
```

```suggestion
    hash = "sha256-dKHb6GTp5aJPuWWXI4ZYnhgdXs23gGWyPymGPGwxcLY=";
```

When possible, convert the SHA256 hashes to SRI format using the command 

`nix hash to-sri sha256:<hash>`

and pasting the result